src/10-deep-strong-coupling.md

```diff
@@ -0,0 +1,1132 @@
+> It should be noted that when the coupling becomes ultra-strong, the equivalent classical pendulum frequency departs from $E_i/\hbar$ significantly. This means that although there is a formal mathematical equivalence between the the quantum and and classical systems, the classical mapping becomes more difficult to interpret the stronger the coupling gets.
+
+The Rabi probability oscillations that we've observed throughout this tutorial series are a signal that we've been working in the "strong coupling" regime. Indeed, we've not included any quantum dissipation such as dephasing or decoherence in our models ($\gamma_{\rm diss}=0$) and so $\gamma_{\rm diss} \ll \omega_{\rm coupling} \ll \omega_1,\omega_2$ has always been satisfied.
+
+Now we're going to consider the deep strong coupling regime of the Rabi model.
+<!-- #endregion -->
+
+## 10.3 Vacuum field deep strong coupling ($n=0$)
+
+
+Much like the classical pendulum, the dynamics of the the Rabi model depend on the relative sizes of the different frequencies/energies in the system, which are represented by $\Delta E, \hbar\omega, U$ in the Hamilonian:
+
+
+$$H = \frac{\Delta E}{2} \sigma_z + \hbar\omega\left(a^{\dagger}a +\frac{1}{2}\right) + U\left( a^{\dagger} + a \right)(\sigma_+ + \sigma_-)$$
+
+
+The deep strong coupling regime, as first explored by [Casanova et al. in 2010](http://dx.doi.org/10.1103/PhysRevLett.105.263603), is then defined as:
+
+$$
+\Delta E , \hbar \omega \lesssim U
+$$
+
+Casanova first focussed on the case when the TLS is degenerate, so that $\Delta E=0$. We'll follow their lead in order to reduce complexity in these initial stages of exploration. In this case, the condition for deep strong coupling reduces to:
+
+$$
+1 \lesssim \frac{U}{\hbar\omega}
+$$
+
+Before we start simulating, let's see if we can predict what will happen.
+
+
+Ideally, we'd like to appeal to our classical pendulum picture, but this is challenging because the number of states (aka quantum pendulums) that are involved. Each state of the system is described by combining the two states of the TLS (denoted $\pm$) with the boson states (denoted by the number of quanta $n$). There are an infinite number of such states $|n, \pm\rangle$.
+
+Thnking about infinity is hard, so let's begin with thinking about two states and see how far we get.
+
+Let's imagine we start the system in the ground state of the TLS and in vacuum - in other words $\psi_0 = |0,-\rangle$. Because of parity conservation, we know that this state is only coupled to states that simultaneously flip the TLS and shift the boson number by one. The only state that satisfies this is $|1,+\rangle$. The two states have different energies/frequencies from one another and so the pendulum set-up looks like this.
+
+![](./img/pendulums-dsc-01.png)
```

Comment:
I'm a bit confused by this figure. It looks like the left pendulum is excited but doesn't state "-" denote ground state?

Reply:
Thank you, that clarified things